### DISCUSSION TITLE
这是对原版文档的翻译+部署吗

### DISCUSSION THREAD

**Discussion by linonetwo** [BramChen/tw5-docs#2] [Q&A] [answered]
看到您在 tiddlywiki5 的仓库里也提交了翻译 https://github.com/Jermolene/TiddlyWiki5/pull/2466 这样

然后您是将翻译后的 tiddlywiki5 单独部署出来吗？上面有没有其他教程内容？是这样的，我们准备做一个共创的中文教程站，里面就不放原版的基础教程了，应该是放一些新手向的、操作流程形式的教程，然后使用 nodejs 版多 *.tid 文件的 wiki 来共创，以免发生 git 冲突等等。

应该不会与您现在做的这个翻译站有功能上的冲突吧？

**Comment by BramChen** [comment, marked as answer]
目前，個人在 [ Jermolene/TiddlyWiki5](https://github.com/Jermolene/TiddlyWiki5) 的提交，主要是語言檔的繁體與簡體中文翻譯。

而這裡的文件內容，則為 https://tiddlywiki.com 原文件的翻譯。文件數量可觀，大多以 Google 翻譯 .tid 內容為繁體中文後，再使用 OpenCC 轉譯為簡體中文。

>操作流程形式的教程

我喜歡這個。
雖然 https://tiddlywiki.com 的內容，從入門到進階的指引已經非常豐富，但對於一般新的使用者而言，似乎親和力還是不足，逐步教學可能才是他們想要的。

> 应该不会与您现在做的这个翻译站有功能上的冲突吧？

當然不會啊。

**Reply by linonetwo** [reply]
那好的，我们最近在 https://github.com/tiddly-gittly/TiddlyWiki-Chinese-Tutorial 建了个站点，可以比较随意地加入自己常用的流程经验，正体简体中文皆可，也欢迎你有空来分享你的经验~
